### DISCUSSION TITLE
Change color of disabled field / form

### DISCUSSION THREAD

**Discussion by mattiason** [Budibase/budibase#3591] [Help]
Is it possible to change the background colour of a field then the field / form is disabled? I would like lighter background colour and sharper text colour. I want to use it for a view-only form.

**Comment by mjashanks** [comment]
There is no way to do this out of the box. You can use the `Edit Custom CSS` to set the background colour of an input, by changing the value of CSS variables.

`--spectrum-textfield-m-background-color-disabled: red;`

You can see what variables are used by inspecting the input element in dev tools.
